openshift/patches/remove_channel_dispatcher_limits.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/data-plane/config/channel/500-dispatcher.yaml b/data-plane/config/channel/500-dispatcher.yaml
+index cd0ef05e1..cd853c25f 100644
+--- a/data-plane/config/channel/500-dispatcher.yaml
++++ b/data-plane/config/channel/500-dispatcher.yaml
+@@ -128,9 +128,6 @@ spec:
+               cpu: 1000m
+               # 600Mi for virtual replicas + 100Mi overhead
+               memory: 700Mi
+-            limits:
+-              cpu: 2000m
+-              memory: 1000Mi
```

Comment:
We need to also patch the Java config https://github.com/openshift-knative/eventing-kafka-broker/blob/main/openshift/patches/remove_broker_dispatcher_limits.patch